avm/res/cache/redis/main.bicep

```diff
@@ -303,6 +304,17 @@ module redis_privateEndpoints 'br/public:avm/res/network/private-endpoint:0.4.1'
   }
 ]
 
+module redis_geoReplication 'linked-servers/main.bicep' = if (!empty(geoReplicationObject)) {
+  name: '${uniqueString(deployment().name, location)}-redis-LinkedServer'
+  params: {
+    redisCacheName: redis.name
+    name: geoReplicationObject.secondaryRedisCacheName
```

Comment:
Should ask for the name as per the parameter.

Reply:
Doesn't it already align with the submodule params? I might be missing the ask here.

Reply:
What's left to the `:` is, yes. I'm more concerened with the right side 😉 

To be more precise, I'd suggest to change
`name: geoReplicationObject.secondaryRedisCacheName`  to
`name: geoReplicationObject.name`

Reply:
Updated.